plugins/LilyPad/Config.h

```diff
@@ -21,13 +21,13 @@
 extern u8 ps2e;
 
 enum PadType {
+    // Changing the number of PadTypes requires numPadTypes in InputManager.h to be modified as well.
```

Comment:
Yes, I thought of that as well. But the use of fake enums is usually frowned upon. :p